.github/linters/.markdown-lint.yml

```diff
@@ -4,11 +4,11 @@ default: false # includes/excludes all rules by default
 MD001: true
 
 # Heading style <https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md003>
-MD003: true
+MD003:
```

Comment:
If I understand this correctly, the `default: false` turns off execution by default but if there's a `style` specified it turns the rule execution on without needing an explicit `true` flag. Is that correct?

If yes, can you please copy across these same config changes over to https://github.com/cheqd/.github/tree/main/workflow-templates so that we have it available consistently across all repos to copy from?